yb-voyager/src/dbzm/yb_cdc_client.go

```diff
@@ -159,6 +160,29 @@ func (ybc *YugabyteDBCDCClient) ListMastersNodes() (string, error) {
 	return masterAddresses, nil
 }
 
+func (ybc *YugabyteDBCDCClient) GetNumOfReplicationStreams() (int, error) {
+	tserverPort := "9100" //TODO: make it internally handled by yb-client
+	if os.Getenv("YB_TSERVER_PORT") != "" {
+		tserverPort = os.Getenv("YB_TSERVER_PORT")
+	}
+	args := fmt.Sprintf("-get_num_of_cdc_streams -master_addresses %s -tserver_port %s", ybc.ybServers, tserverPort)
+
+	if ybc.sslRootCert != "" {
+		args += fmt.Sprintf(" -ssl_cert_file '%s'", ybc.sslRootCert)
+	}
+
+	stdout, err := ybc.runCommand(args)
+	if err != nil {
+		return 0, fmt.Errorf("running command with args: %s, error: %s", args, err)
+	}
+	//stdout - Streams: <num_of_streams>
+	numOfStreams, err := strconv.Atoi(strings.Trim(strings.Split(stdout, ": ")[1], " \n"))
```

Comment:
we should check first that the output of split is a non-empty slice. Directly indexing the array `[1]` can lead to a panic